<launch>
    <arg name="annotation_mapping_file"
         default="/home/mark/Dropbox/Developer/mark-VCLA-repos/OpenBottle/grammar_induction/annotation_mapping.txt" />
    <arg name="sampler_dir"
         default="/home/mark/Dropbox/Developer/mark-VCLA-repos/OpenBottle/grammar_induction/" />
    <arg name="grammar_file"
         default="
/home/mark/Desktop/Mass/IROS2017/datacollection/OldDesktop/tf_convert_merged_annotations/all_grasp_right_added_xopenbottle/grammar.txt" />
    <arg name="sample_file"
         default="/home/mark/Dropbox/Developer/mark-VCLA-repos/OpenBottle/grammar_induction/sample.txt" />
    <node name="sentence_srv" pkg="sentence_srv" type="sentence_srv_node.py" output="screen">SIMPLEX/
        <param name="annotation_mapping_file" type="string" value="$(arg annotation_mapping_file)" />SIMPLEX/
        <param name="sampler_dir" type="string" value="$(arg sampler_dir)" />
        <param name="grammar_file" type="string" value="$(arg grammar_file)" />
        <param name="sample_file" type="string" value="$(arg sample_file)" />
    </node>
</launch>